male-seed-beetle/scripts/download-data.py

```diff
@@ -0,0 +1,34 @@
+from pathlib import Path
+
+import requests
+
+#testing:
+url = 'https://datadryad.org/stash/downloads/file_stream/30903'
+
+def download_data(url: str):
```

Comment:
This function actually doesn't contain any code. Is that intentional?

Reply:
No, those aren't part of the function. They would need to be indented to the level of the function for them to be part of it. I think it's fine if the script doesn't have a function in it.